src/components/admin/AddContent.tsx

```diff
@@ -2,43 +2,142 @@
 import { useEffect, useState } from 'react';
 import { AddNotionMetadata } from './AddNotionMetadata';
 import { Input } from '../ui/input';
+import DiscordService from '@/app/service/DiscordService';
+import { useRecoilState } from 'recoil';
+import { Checkbox } from '../ui/checkbox';
+import { loader } from '@/store/atoms/loader';
 
 export const AddContent = ({
+  rest,
   courseId,
   parentContentId,
+  courseTitle,
 }: {
+  rest: string[];
   courseId: number;
   parentContentId?: number;
+  courseTitle: string;
 }) => {
   const [type, setType] = useState('folder');
   const [imageUri, setImageUri] = useState('');
   const [title, setTitle] = useState('');
   const [metadata, setMetadata] = useState({});
+  const [checked, setChecked] = useState<Boolean>(true);
   const [adminPassword, setAdminPassword] = useState('');
+  const [loading, setLoading] = useRecoilState(loader);
+
+  const { sendUpdateToDiscord } = DiscordService();
+
+  interface DiscordData {
+    type: string;
+    thumbnail: string;
+    title: string;
+    courseTitle: string;
+    courseId: number;
+    currFolderId: number;
+    mediaId: number;
+  }
+
+  const postOnDiscord = (data: DiscordData) => {
+    sendUpdateToDiscord(data);
+  };
+
+  const handleContentSubmit = async () => {
+    const fieldsToCheck = [
+      'video_1080p_0',
+      'video_720p_0',
+      'video_360p_0',
+      'notionId',
+    ];
+
+    setLoading(true);
+
+    const response = await fetch('/api/admin/content', {
+      body: JSON.stringify({
+        type,
+        description: '',
+        thumbnail: imageUri,
+        title,
+        courseId,
+        parentContentId,
+        metadata,
+        adminPassword,
+      }),
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    const respData: any = await response.json();
+
+    const data = {
+      type,
+      thumbnail: imageUri,
```

Comment:
hardcode this to https://d2szwvl7yo497w.cloudfront.net/courseThumbnails/video.png

Reply:
@hkirat , Please see #601 , I have raised a new PR and resolved all of the changes you suggested in live